ActindoWorkFlow/UserInterface/03a_Processes.md

```diff
@@ -0,0 +1,634 @@
+Click the drop-down list to display a specific workflow type. The workflows created in the *Workflows* menu entry are listed here. For detailed information about creating a workflow, see [Create a workflow](/ActindoWorkFlow/Operation/01_ManageWorkflows.md#create-a-workflow).
+
+
+
+- ![Search](/Assets/Icons/Search.png "[Search]") (Search)  
+    Click this button to display the search bar and search for a workflow.
+
+- ![Refresh](/Assets/Icons/Refresh01.png "[Refresh]") (Refresh)  
+    Click this button to update the list of workflows.
+
+- ![Columns](/Assets/Icons/Columns.png "[Columns]") Columns (x)  
+    Click this button to display the columns bar and customize the displayed columns and the order of columns in the list. The x indicates the number of columns that are currently displayed in the list.
+
+- ![Filter](/Assets/Icons/Filter.png "[Filter]") Filter (x)  
+    Click this button to display the filter bar and customize the active filters. The x indicates the number of filters that are currently active.
+
+
+The list displays all processes. Depending on the settings, the displayed columns may vary. All fields are read-only.
+
+  - [x]  
+  Select the checkbox to display the editing toolbar. If you click the checkbox in the header, all processes in the list are selected.
+
+  Depending of the process status, the editing toolbar may display different buttons.
+
+  - ![Abort](/Assets/Icons/Stop.png "[Abort]") (Abort)  
+    Click this button to end the process. This button is displayed when the process status is *Error* or *In Progress*. When clicking this button, the process status changes to *Aborted*.
+  - ![Suspend](/Assets/Icons/Pause.png "[Suspend]") (Suspend)  
+    Click this button to suspend the process temporarily. This button is displayed when the process status is *In Progress*.  When clicking this button, the process status changes to *Suspended*.
+  - ![Start](/Assets/Icons/Play.png "[Start]") (Start)  
+    Click this button to resume the process. This button is displayed when the process status is *Suspended*. When clicking this button, process status changes again to *In Progress*.
+
+
+  - *Status*  
+  Status of the process. The different process statuses are displayed in different colours. The following statuses are possible:
+
+    - *Ready*
+    - *In Progress*
+    - *Error*
+    - *Done*
+    - *Dead*
+    - *Suspended*
+    - *Aborted*
+
+[comment]: <> (Repeat or leave it out? Add colour code?)
+
+  - *Suspended Actions*  
+  If the process contains suspended actions, a ![Warning](/Assets/Icons/Warning.png "[Warning]") sign is displayed. Click the process to display the individual actions, see [Actions](#actions).
+
+  - *Subject*  
+  Process description.
+
+  - *Workflow*  
+  Name of the workflow.
+
+  - *Queue Type*  
+  Queue type configured in the workflow. For detailed information about configuring the queue types, see [Configure the queue types](ActindoWorkFlow/Integration/01_ConfigureQueueTypes.md).
```

Comment:
interessante spalte. Hier würde ich oli gern fragen, ob es auch ne andere option gibt außer "according to configuration in workflow". Ist ja sonst ziemlich sinnlos diese spalte...

Reply:
Kommentar hinzugefügt